journal/src/Views/ViewConverter/Blocks/eLifeFrontContentHeaderConverter.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Libero\Journal\Views\ViewConverter\Blocks;
+
+use Libero\Dom\Element;
+use Libero\Dom\Node;
+use Libero\PatternsBundle\Views\ViewConverter\LangAttributes;
+use Libero\Views\View;
+use Libero\Views\ViewConverter;
+use Symfony\Component\Translation\TranslatorInterface;
+use function array_filter;
+use function Functional\map;
+use const Libero\PatternsBundle\LIBERO;
+
+final class eLifeFrontContentHeaderConverter implements ViewConverter
+{
+    use LangAttributes;
+
+    private $converter;
+    private $translator;
+
+    public function __construct(ViewConverter $converter, TranslatorInterface $translator)
+    {
+        $this->converter = $converter;
+        $this->translator = $translator;
+    }
+
+    public function convert(Node $object, ?string $template, array $context = []) : ?View
+    {
+        if (!$object instanceof Element || '@LiberoPatterns/patterns/content-header.html.twig' !== $template || 'front' !== $object->getName() || LIBERO !== $object->getNamespace()) {
+            return null;
+        }
+
+        $context['level'] = ($context['level'] ?? 1) + 1;
+
+        return new View(
+            $template, array_filter(
+                [
+                    'digest' => $this->digest($object, $context),
+                ]
+            )
+        );
+    }
+
+    private function digest(Element $object, array $context = []) : ?array
+    {
+        $digestElement = $object->get('elife:digest');
+
+        if (!$digestElement instanceof Element) {
+            return null;
+        }
+
+        return [
```

Comment:
this looks like a generic view model object, maybe an extension to `ArrayObject` could help it be both array-like and add some validation (for standard fields like `attributes`) or behavior (utility methods that suddenly have a place to live?)